src/Components/App.js

```diff
@@ -6,13 +6,11 @@ import { useSelector } from "react-redux";
 //Page Components
 import MyBatches from "./batches/MyBatches";
 import LoginPage from "./login/LoginPage";
-import LogoutPage from "./login/LogoutPage";
```

Comment:
Is there a reason for removing the LogoutPage?

Reply:
We don't need LogoutPage. When logging out, the user should go back to the homepage

Reply:
That sounds good to me.